### PR TITLE
fix golint issue for pkg/volume/flexvolume package

### PR DESCRIPTION
**Pull request by leakingtapan**
**What this PR does / why we need it**:
fix golint issues for pkg/volume/flexvolume

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Ref #68026

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/volume/flexvolume/driver-call.go [hunk 1]
```diff
@@ -219,6 +223,7 @@ type DriverStatus struct {
 	Capabilities *DriverCapabilities `json:",omitempty"`
 }
 
+// DriverCapabilities represents what driver an do
```
Comment (chakri-nelluri) [inline pkg/volume/flexvolume/driver-call.go]: *driver can do

**Comment by chakri-nelluri** [comment]
Thanks @leakingtapan. Please fix the nit.. and we are good to go.

File: pkg/volume/flexvolume/driver-call.go (hunk 1 above)
Comment (leakingtapan) [reply pkg/volume/flexvolume/driver-call.go]: Fixed. @chakri-nelluri  could you give a ok-to-test, so that bot can start testing? thx

**Review by chakri-nelluri — APPROVED** [review]

**Comment by nikhita** [comment]
/assign @dims 
for hack/ approval

**Comment by dims** [comment]
/approve

thanks @leakingtapan

**Pull request merged**